webapp/params.go

```diff
@@ -190,3 +190,30 @@ func ParsePathsParam(r *http.Request) (paths mapset.Set) {
 	}
 	return paths
 }
+
+// ParseLabelsParam returns a set list of test-run labels to include, or nil if
+// no labels are provided.
+func ParseLabelsParam(r *http.Request) (labels mapset.Set) {
+	labelParams := r.URL.Query()["label"]
+	labelsParam := r.URL.Query().Get("labels")
```

Comment:
Can you exercise labels=foo,bar in tests as well? And label=foo&label=bar if that is intended to work, which it seems like it is. (Not testing labels=foo,bar&labels=baz seems fine.)

Reply:
Done.